### DISCUSSION TITLE
注意：使用沉浸式翻译的 Prompt 会导致 Gemini API 被封禁（报429错误）

### DISCUSSION THREAD

**Discussion by yjmp14** [immersive-translate/immersive-translate#1561] [Help]
### 概述 | Summary

使用该插件的 Prompt 会收到 429 错误，换一个 Prompt 就正常，经进一步研究，Gooogle 认为使用该 Prompt 是`似乎通过将一个项目分为多个项目的方式来规避我们的配额限制`，建议大家自行修改 Gemini 模型的 Prompt，以免 API 被 Google 封禁。

> 我们检测到“Generative Language Client”(gen-lang-client-xxxxxxxxxx) 违反了服务条款
> 此项目似乎通过将一个项目分为多个项目的方式来规避我们的配额限制。
> 这违反了 [Google Cloud Platform 服务条款 ](https://console.developers.google.com/terms)（包括适用的每一项 Google API [服务条款 ](https://console.developers.google.com/terms)）。
> 若不采取措施并提交以下表单，您的项目可能会被暂停：

**Comment by Nicecsh** [comment]
我的Gemini刚刚被封禁了，搞得一头雾水，不知道什么原因。看了你的贴子才知道是跟使用沉浸式翻译有关😥。
麻烦问一下有改好的Prompt可以参考吗？

**Comment by crybest** [comment]
那么问题来了，怎么改？

**Reply by jlong7398** [reply]
我是直接把通用提示词改成中文的，然后润色一下就没有遇到这种情况了。

**Comment by YANANEER** [comment]
我付费key因为这个直接给我项目封了，不知道谷歌在搞什么
